src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--day-fri calendar--month-length-30">
+      <div class="calendar__offset"></div>
+      <div class="calendar__day calendar__day--1"></div>
+      <div class="calendar__day calendar__day--2"></div>
+      <div class="calendar__day calendar__day--3"></div>
+      <div class="calendar__day calendar__day--4"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```
There is no need to add additional classes to the calendar elements. 
You can get the needed one using pseudoclasses